src/patternfly/components/Wizard/wizard.scss

```diff
@@ -39,6 +39,13 @@
     --pf-c-wizard__close--Right: var(--pf-c-wizard__close--lg--Right);
   }
 
+  // Cancel
+  --pf-c-wizard__footer__link--MarginLeft: var(--pf-global--spacer--lg);
```

Comment:
I don't think we use two __ separators, do we? Instead I think I'd use `--pf-c-wizard__footer-link` (and so forth throughout)